src/ToastNotification/ToastNotificationContainer.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace PackagedUi\Fusion\ToastNotification;
+
+use Packaged\Glimpse\Tags\Div;
+use Packaged\Ui\Html\HtmlElement;
+use PackagedUi\BemComponent\BemComponentTrait;
+use PackagedUi\Fusion\Component;
+use PackagedUi\Fusion\ComponentTrait;
+
+class ToastNotificationContainer extends Div implements Component
+{
+  use BemComponentTrait;
+  use ComponentTrait;
+
+  /** @var ToastNotification[] */
+  protected $_toastNotifications = [];
+
+  /** @var int */
+  protected $_delay = 5000;
+
+  /** @var ToastNotificationPosition */
+  protected $_position;
+
+  /**
+   * @inheritDoc
+   */
+  public function getBlockName(): string
+  {
+    return 'toast-notification-container';
+  }
+
+  /**
+   * @param ToastNotification $toastNotification
+   *
+   * @return ToastNotificationContainer
+   */
+  public function addToastNotification(ToastNotification $toastNotification): ToastNotificationContainer
+  {
+    $this->_toastNotifications[] = $toastNotification;
+    return $this;
+  }
+
+  /**
+   * @param int $delay
+   *
+   * @return ToastNotificationContainer
+   */
+  public function setDelay(int $delay): ToastNotificationContainer
```

Comment:
whats this a delay for?

Reply:
The delay on the toast notification. The container is what controls the delay as we have to use nth type and a for loop to add a transition delay so they don't all appear at once